.circleci/config.yml

```diff
@@ -38,6 +38,21 @@ commands:
           root: *workspace_root
           paths:
             - .aws
+  assume-role-and-persist-workspace-mosaic-production:
+    description: "Assumes deployment role and persists credentials across jobs"
```

Comment:
I think this uses the same description as the other `assume-role-and-persist-workspace` command. Wondering if it's worth mentioning which jobs these credentials would be persisted across 🤔

Reply:
Ah yeah, I like that idea. I'll update to `Assumes deployment role and persists credentials across jobs for Mosaic-Production`.

Reply:
Updated in https://github.com/LBHackney-IT/social-care-case-viewer-api/pull/165/commits/189a02d8dda8cb793ed64de994649484bc18a48c.